tests/platform_helper/test_command_notify.py

```diff
@@ -0,0 +1,261 @@
+import unittest
+from unittest.mock import patch
+
+from click.testing import CliRunner
+
+from dbt_platform_helper.commands.notify import add_comment
+from dbt_platform_helper.commands.notify import environment_progress
+from dbt_platform_helper.commands.notify import get_build_url
+
+
+@patch("dbt_platform_helper.commands.notify._get_slack_client")
+class TestNotify(unittest.TestCase):
+    def test_getting_build_url(self, webclient):
+        self.assertEqual(
+            get_build_url("arn:aws:codebuild:region:000000000000:build/project:example-build-id"),
+            "https://region.console.aws.amazon.com/codesuite/codebuild/000000000000"
+            "/projects/project/build/project%3Aexample-build-id",
+        )
+
+    def test_sending_progress_updates_with_no_optional_elements(self, webclient):
```

Comment:
Is it possible to parametrize some of these progress updates tests?

Reply:
I had thought about it, but thought it'd get messy. I'll take a look and see if I can do it neatly.